recipes/default.rb

```diff
@@ -15,3 +15,154 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# Install PHP + Modules
+node.default['php']['version'] = '7.4'
+node.default['osl-php']['use_ius'] = true
+node.default['osl-php']['use_opcache'] = true
+node.default['osl-php']['php_packages'] =
+  %w(
+    fpm
+    cli
+    imap
+    json
+    mysqlnd
+    mbstring
+  )
+
+include_recipe 'osl-php'
+include_recipe 'osl-apache'
+include_recipe 'osl-apache::mod_php'
+include_recipe 'osl-mysql::client'
+
+# Certificate (wildcard is placeholder for now)
+cert_path = '/etc/pki/tls/certs/wildcard.pem'
+key_path = '/etc/pki/tls/certs/wildcard.key'
+
+# Following this tutorial: https://linuxize.com/post/set-up-an-email-server-with-postfixadmin/
+
+group 'vmail' do
+  gid 5000
+end
+
+user 'vmail' do
+  comment 'Owner of all mailboxes. Used to access emails on this server'
+  home    '/var/mail/vmail'
+  group   'vmail'
+  shell   '/usr/sbin/nologin'
+end
```

Comment:
Let's not set the `gid` value and also use `system true` for both the group and user.